tests/conftest.py

```diff
@@ -393,6 +393,14 @@ def moving_image(remote_sample) -> pathlib.Path:
     return remote_sample("moving_image")
 
 
+@pytest.fixture(scope="session")
+def dfbr_features(remote_sample) -> pathlib.Path:
+    """Sample pytest fixture for DFBR features.
+    Download features for pytest.
+    """
+    return remote_sample("dfbr_features")
+
+
 @pytest.fixture(scope="session")
 def fixed_mask(remote_sample) -> pathlib.Path:
     """Sample pytest fixture for fixed mask.
```

Comment:
Docstring new lines missing here.

Reply:
Added in other places as well. 
https://github.com/TissueImageAnalytics/tiatoolbox/blob/4a4290bd4a06835ac1f8e333f4396e764e0555a8/tests/conftest.py#L381-L386